apps/expo-go/README.md

```diff
@@ -0,0 +1,40 @@
+# Expo Go
+
+For external contributors: If you want to contribute to the Expo SDK, use the Bare Expo app for developing and testing your changes (unless your changes are specific to the Expo Go app).
+
+## Building Expo Go
+
+1. Build React Native
```

Comment:
mabe restructure the readme to separate ios and android builds, given the section is `Building Expo Go` but you are focusing on android build

Reply:
added more info for iOS